spec/features/idv/analytics_spec.rb

```diff
@@ -28,6 +28,17 @@
       timed_out: false,
       transaction_id: 'ddp-mock-transaction-id-123' }
   end
+  let(:base_proofing_components) do
+    # rubocop:disable Layout/LineLength
+    { document_check: 'mock', document_type: 'state_id', source_check: 'aamva', resolution_check: 'lexis_nexis', threatmetrix: threatmetrix, threatmetrix_review_status: 'pass' }
+    # rubocop:enable Layout/LineLength
+  end
+  let(:lexis_nexis_address_proofing_components) do
+    base_proofing_components.merge(address_check: 'lexis_nexis_address')
```

Comment:
I thought this was more expressive than repeating the whole thing, even if it's probably a few CPU cycles slower.